pnpm-workspace.yaml

```
@@ -2,7 +2,9 @@ packages:
   # all packages in direct subdirs of packages/
   - 'packages/ai-core'
   - 'packages/gen-ai-hub'
+  # sample code
+  - 'sample-code'
```

```suggestion
  - '!sample-code'
```

So we don't publish it

---

But wouldn't this also exclude the package from the build process?

---

I think it would. I will take care of this on my canary release branch.